java/model/src/main/java/com/instaclustr/cassandra/sidecar/model/result/BackupResult.java

```diff
@@ -0,0 +1,21 @@
+package com.instaclustr.cassandra.sidecar.model.result;
+
+import java.util.UUID;
+
+import com.instaclustr.cassandra.sidecar.model.operation.OperationType;
+
+public class BackupResult extends OperationResult {
+
+    public BackupResult(final UUID id) {
+        this(OperationType.BACKUP, id);
+    }
+
+    public BackupResult(final OperationType type, final UUID id) {
+        super(type, id);
+    }
+
+    @Override
+    public String toString() {
+        return "BackupResult{} " + super.toString();
```

Comment:
Look into using Guava's `MoreObjects.toStringHelper(...)` function, to assist with this and keep it clean.